runtime/cni_integ_test.go

```diff
@@ -209,8 +355,51 @@ func writeCNIConf(path string, networkName string, nameserver string) error {
       "dns": {"nameservers": ["%s"]}
     },
     {
-      "type": "tc-redirect-tap"
+      "type": "%s"
     }
   ]
-}`, networkName, nameserver)), 0644)
+}`, networkName, nameserver, chainedPluginName)), 0644)
+}
+
+func runCommand(t *testing.T, name string, args ...string) {
+	output, err := exec.Command(name, args...).CombinedOutput()
+	require.NoErrorf(t, err, "command %+v failed, output: %s", append([]string{name}, args...), string(output))
+}
+
+func perfTestVMCount(t *testing.T) int {
```

Comment:
This might be overkill but I think you could make three of these simpler:

```go
func perTestVMCount(t testing.T) int {
	t.Helper()
	return getEnvVarInt(t, "PERF_VMCOUNT")
}

func getEnvVarInt(t *testing.T, key string) int {
	t.Helper()
	envVal := os.Getenv(key)
	val, err := strconv.Atoi(envVal)
	require.NoErrorf(t, err, "%s env is not set or is not an int: %q", envKey, envVal)
	return val
}
```

Reply:
Yeah it was easier for me to copy-paste before but it's reached the point where that logic is duped enough to have a helper, will update.